hazelcast/src/main/java/com/hazelcast/util/VersionCheck.java

```diff
@@ -116,23 +120,32 @@ private void doCheck(Node hazelcastNode, String version, boolean isEnterprise) {
                 ? Long.toString(MemoryUnit.BYTES.toGigaBytes(totalNativeMemorySize)) : "0";
         //Calculate connected clients to the cluster.
         Map<ClientType, Integer> clusterClientStats = hazelcastNode.clientEngine.getConnectedClientStats();
+        RuntimeMXBean runtimeMxBean = ManagementFactory.getRuntimeMXBean();
+
+        Long clusterUpTime = hazelcastNode.getClusterService().getClusterClock().getClusterTime()
+                - hazelcastNode.getClusterService().getClusterClock().getClusterStartTime();
+        String clusterId = hazelcastNode.getConfig().getGroupConfig().getName()
```

Comment:
Yes, you are right. I have added UUID based id for the cluster and getting it from ClusterService.